src/sql/workbench/services/fileBrowser/browser/urlBrowserDialog.ts

```diff
@@ -0,0 +1,477 @@
+	private _storageAccounts: AzureGraphResource[];
+	private _selectedStorageAccount: AzureGraphResource;
+	private _blobContainers: BlobContainer[];
+	private _selectedBlobContainer: BlobContainer;
+	private _backupFiles: Blob[];
+
+	private _ownerUri: string;
+	private _restoreDialog: boolean;
+	private _body: HTMLElement;
+	private _accountSelectorBox: SelectBox;
+	private _tenantSelectorBox: SelectBox;
+	private _subscriptionSelectorBox: SelectBox;
+	private _storageAccountSelectorBox: SelectBox;
+	private _blobContainerSelectorBox: SelectBox;
+	private _sasInputBox: InputBox;
+	private _sasButton: Button;
+	private _backupFileInputBox: InputBox;
+	private _backupFileSelectorBox: SelectBox;
+	private _okButton: Button;
+	private _cancelButton: Button;
+	private _onOk = new Emitter<string>();
+	public onOk: Event<string> = this._onOk.event;
+
+
+	constructor(title: string,
+		restoreDialog: boolean,
+		defaultBackupName: string,
+		@ILayoutService layoutService: ILayoutService,
+		@IThemeService themeService: IThemeService,
+		@IContextViewService private _contextViewService: IContextViewService,
+		@IAdsTelemetryService telemetryService: IAdsTelemetryService,
+		@IContextKeyService contextKeyService: IContextKeyService,
+		@IClipboardService clipboardService: IClipboardService,
+		@ILogService logService: ILogService,
+		@ITextResourcePropertiesService textResourcePropertiesService: ITextResourcePropertiesService,
+		@IAccountManagementService private _accountManagementService: IAccountManagementService,
+		@IAzureAccountService private _azureAccountService: IAzureAccountService,
+		@IAzureBlobService private _blobService: IAzureBlobService,
+		@IInstantiationService private _instantiationService: IInstantiationService
+	) {
+		super(title, TelemetryKeys.ModalDialogName.FileBrowser, telemetryService, layoutService, clipboardService, themeService, logService, textResourcePropertiesService, contextKeyService, { dialogStyle: 'flyout', hasTitleIcon: false, hasBackButton: true, hasSpinner: true });
+		this._defaultBackupName = defaultBackupName;
+		this._restoreDialog = restoreDialog;
+	}
+
+	protected layout(height?: number): void {
+	}
+
+	protected renderBody(container: HTMLElement) {
+		this._body = DOM.append(container, DOM.$('.url-browser-dialog'));
+	}
+
+	public override render() {
+		super.render();
+		attachModalDialogStyler(this, this._themeService);
+
+		if (this.backButton) {
+
+			this._register(this.backButton.onDidClick(() => {
+				this.close();
+			}));
+
+			this._register(attachButtonStyler(this.backButton, this._themeService, { buttonBackground: SIDE_BAR_BACKGROUND, buttonHoverBackground: SIDE_BAR_BACKGROUND }));
+		}
+
+		let tableContainer: HTMLElement = DOM.append(DOM.append(this._body, DOM.$('.option-section')), DOM.$('table.url-table-content'));
+		tableContainer.setAttribute('role', 'presentation');
+
+		let azureAccountLabel = localize('azurebrowser.account', "Azure Account*");
```

Comment:
What's the * for? If this is a required field you should be setting the required property field, not adding it directly to the string